.flake8

```diff
@@ -1,2 +0,0 @@
-[flake8]
```

Comment:
This file is used by the backend, maybe move to backend folder instead and change loading path in docker-compose?

Reply:
There were 2 copies of it already. Added the right path to docker-compose - great catch!